### PR TITLE
Homework lesson 14 classes , own project - status in progres

### PR DESCRIPTION
**Pull request by sorinmosut**

### CODE REVIEW

File: lesson_14/mosut_sorin/hw_lesson14_operators_over_mosut_sorin.py [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+car_10 = WorkOrder('mm15ury', 'uu125478541235478', 'dacia', '1310', 'Nicolae Mosut')
+work_orders_dict.update(WorkOrdersDict(car_10))
+
+print_open_wo() # check
+
+list_for_closing = [
+    ['200503001', 60, 1500],
+    ['200507002', 110, 20000],
+    ['200508003', 3, 800],
+    ['200511004', 2.5,600],
```
Comment (sorinmosut) [inline lesson_14/mosut_sorin/hw_lesson14_operators_over_mosut_sorin.py:327]: For test the functionality , change inside the string , '11' with today date , do this for  all futher examples , but only those with '11' , on position [4:5].

File: lesson_14/mosut_sorin/assignement_mosut_sorin.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+features:
+- opening a work order
+- add the new work order in open work orders database
+- close a work order , delete from open work orders database and add to closed work orders database
+- ensure that all the necessary data are in when a work order is closed
+- ensure that any work order closed can not be deleted
+- display all the open / closed work orders 
+- check if a car has open / closed work orders (by registration number) , display it / them , and calculate all costs
+- check if a client has open / closed work orders (by registration number) , display it / them , and calculate all costs
+<!--stackedit_data:
```
Comment (Liitle) [inline lesson_14/mosut_sorin/assignement_mosut_sorin.md:45]: You can remove those 3 lines :smile:

**Review by Liitle — COMMENTED** [review]
Nice work, really complex challenge. @sorinmosut do you have specific question about the implementation? There are places where we can do some refactoring to make the code cleaner, but it will be easyer if you woud identify what areas you want to improve or that you have questions about

**Review by sorinmosut — COMMENTED** [review]
> Nice work, really complex challenge. @sorinmosut do you have specific question about the implementation? There are places where we can do some refactoring to make the code cleaner, but it will be easyer if you woud identify what areas you want to improve or that you have questions about

Especially if i used pythonic the classes , objects , methods. Becouse i manged to get the right results but i dont know if i do it in the correct way.

**Comment by Liitle** [comment]
@sorinmosut The challenge is correct, I will merge the PR and we can talk about your solution after one of the classes to discuss some clean code possible improvements :smile:

**Comment by sorinmosut** [comment]
Ok , thanx. :)

**Pull request merged**